testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -16099,6 +16099,28 @@ void testTimestampDiff(boolean coercionEnabled) {
         "BOOLEAN", true);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6950">[CALCITE-6950]
+   * Use ANY operator to check if an element exists in an array throws exception</a>. */
+  @Test void testQuantifyCollectionOperators2() {
+    final SqlOperatorFixture f = fixture();
+    QUANTIFY_OPERATORS.forEach(operator -> f.setFor(operator, SqlOperatorFixture.VmName.EXPAND));
+
+    f.checkNull("1.0 = some (ARRAY[2,3,null])");
```

Comment:
why is this null?

Reply:
Because the value of `1.0 == null` is NULL, So the value of the expression is null.

Reply:
So this stops at the first match that returns true.
But if there is no match, what does it return, the last comparison, or the AND of all comparisons?
I.e., what is the output for 1 = some(array[2, NULL, 3])?

Reply:
` 1 = some(array[2, NULL, 3])` will return NULL, I will update the unit test.